gui/default/syncthing/core/syncthingController.js

```diff
@@ -567,8 +567,7 @@ angular.module('syncthing.core')
                 $scope.connectionsTotal = data.total;
 
                 data = data.connections;
-                var oldCache = $scope.remoteGUICache;
-                var newCache = {};
+                var currentAddresses = [];
```

Comment:
You have removed the line where addresses are actually added to the `currentAddresses` array. Also the code comment about asynchronous probing should stay IMO.